protocol/drivermain.go

```diff
@@ -0,0 +1,176 @@
+package protocol
```

Comment:
Ok. Note that this does not build a command itself, but it's used from driver's main.go.